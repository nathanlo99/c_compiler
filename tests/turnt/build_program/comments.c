//
// Program with two integer parameters, a and b
//   returns the sum of a and b
//
int wain(int a, int b) {
  int sum = 0;
  return sum = a + b; // unhelpful comment about summing a and b
}
