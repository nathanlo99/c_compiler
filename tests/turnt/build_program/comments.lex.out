INT (int) at [5:1 - 5:3]
WAIN (wain) at [5:5 - 5:8]
LPAREN (() at [5:9 - 5:9]
INT (int) at [5:10 - 5:12]
ID (a) at [5:14 - 5:14]
COMMA (,) at [5:15 - 5:15]
INT (int) at [5:17 - 5:19]
ID (b) at [5:21 - 5:21]
RPAREN ()) at [5:22 - 5:22]
LBRACE ({) at [5:24 - 5:24]
RETURN (return) at [6:3 - 6:8]
ID (a) at [6:10 - 6:10]
PLUS (+) at [6:12 - 6:12]
ID (b) at [6:14 - 6:14]
SEMI (;) at [6:15 - 6:15]
RBRACE (}) at [7:1 - 7:1]
